site/src/pages/ManagementSettingsPage/IdpSyncPage/IdpSyncPage.tsx

```diff
@@ -84,25 +71,34 @@ export const IdpSyncPage: FC = () => {
 					description="Group and role sync mappings (configured using Coder CLI)."
 					tooltip={<IdpSyncHelpTooltip />}
 				/>
-				<Stack direction="row" spacing={2}>
-					<Button
-						startIcon={<LaunchOutlined />}
-						component="a"
-						href={docs("/admin/auth#group-sync-enterprise")}
-						target="_blank"
-					>
-						Setup IdP Sync
-					</Button>
-				</Stack>
+				<Button
+					startIcon={<LaunchOutlined />}
+					component="a"
+					href={docs("/admin/auth#group-sync-enterprise")}
+					target="_blank"
+				>
+					Setup IdP Sync
+				</Button>
```

Comment:
I decided to hide this button on the provisioners page, since the page is telling the user they're not allowed to use the feature. maybe it's fine since it just links to the docs tho. wdyt?

Reply:
Yes, I just left it because its only a docs button.